### PR TITLE
updated tablesorting

### PR DESCRIPTION
**Pull request by jvanasco**
Here's an approach to a tablesorting fix (https://github.com/Pylons/pyramid_debugtoolbar/issues/148)

* It keeps the existing `pDebugSortable` css class as the trigger.
* `pDebugSortable` is documented in the panel writing docs.
* The sorting library is upgraded to the more modern fork, which should be better compatible with the current jquery and also handles additional sorting methods.
(http://mottie.github.io/tablesorter/docs/example-widget-bootstrap-theme.html)

I included the entirety of the sortable's 'dist' library, although only 3 files are used.  They are documented, along with the version, in a README in that directory.  This should make ongoing maintenance much simpler.

### CODE REVIEW

**Comment by stevepiercy** [comment]
I'm curious, why would you included the full dist instead of only the required files?

**Comment by mmerickel** [comment]
Please remove the full dist and only include the minimized js file.

**Comment by jvanasco** [comment]
@stevepiercy @mmerickel It's a habit from maintaining large projects and doing code reviews.  I'll happily remove it if a second set of eyes could please confirm that I'm implementing the library right (and not simply having everything magically work).

**Comment by mmerickel** [comment]
This looks fine to me offhand but again it needs to be cleaned up.

**Comment by jvanasco** [comment]
If I have time tomorrow I'll address

**Comment by jvanasco** [comment]
This works and doesn't seem to trip any errors on random views.

jquery isn't loaded on the traceback view and can break javascript on some browsers so I nested the sort apply in a try/catch block.

File: pyramid_debugtoolbar/static/js/toolbar.js [hunk 1]
```diff
@@ -59,14 +60,22 @@ $('#settings .switchable').click(function() {
   }
 });
 
-// $(".pDebugSortable").tablesorter();
 
-bootstrap_panels = ['pDebugVersionPanel', 'pDebugHeaderPanel']
+try {
```
Comment (mmerickel) [inline pyramid_debugtoolbar/static/js/toolbar.js]: why would this fail? Please at least add a comment here about where it's expected to fail.

Comment (jvanasco) [inline pyramid_debugtoolbar/static/js/toolbar.js]: updated.  it will only happen on the interactive traceback view.

**Comment by mmerickel** [comment]
Is there any reason jquery is not included on the traceback view? Could we just add it? Which view are you talking about exactly? It looks to me like jquery is loaded inside debugger.js which is loaded from https://github.com/Pylons/pyramid_debugtoolbar/blob/039e8805648142880d3e14e99ab3a683e39edf39/pyramid_debugtoolbar/templates/exception.dbtmako (also in console.dbtmako).

**Comment by mmerickel** [comment]
I think #252 is related to this. It appears jquery is not actually being loaded before toolbar.js properly. We need to clean this up get rid of it now that the toolbar is properly isolated from the user's app.

**Comment by mmerickel** [comment]
Okay I opened #256 to make this a bit more clear. It seems to me we may not need to include toolbar.js in the exception views at all, so that's a thing to consider. I would greatly appreciate some help cleaning this up instead of just wrapping stuff in a try/catch and asking me to maintain it. :-)

**Comment by jvanasco** [comment]
The error can happen in the interactive debugger if an exception is raised.  

On that template/view, the toolbar.js file is pulled in BUT Jquery is not.  I don't know why the toolbar.js is used on that page, and wasn't sure how it was pulled in.  I think it's done within js like you mentioned.  I'm unfamiliar with those components so didn't want to risk breaking things.  

In the current master, if you uncomment the existing tablesorter line, the interactive debugger breaks on most browsers because of no Jquery.  All the icons to expand sections and use the interactive features stop working. It doesn't seem to happen as much with the new library, but it has on a few browsers for me.  

 The trace back panel is unaffected, because Jquery is in the toolbar view.  It's just the interactive debugger that is busted.

Adding Jquery to that template would address the issue.

**Comment by mmerickel** [comment]
It's fine I just went ahead and sorted out the dependencies myself. Thanks for updating the files.

**Pull request merged**